documentation-website/cypress/e2e/middlewares.cy.ts

```diff
@@ -11,7 +11,8 @@ describe('usage/middlewares', () => {
     cy.url().should('eq', BASE_URL + page.middlewares,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Middlewares | Usage | @idrinth/api-bench',);
+    cy.title().should('eq', 'Middlewares | Usage | ' +
```

Comment:
this package will never exist, please revert the change for now

Reply:
Reverted